packages/hawtio/src/plugins/springboot/index.ts

```diff
@@ -0,0 +1,18 @@
+import { hawtio, HawtioPlugin } from '@hawtiosrc/core'
+
+import { pluginId, pluginPath } from './globals'
+import { workspace } from '@hawtiosrc/plugins'
+import { helpRegistry } from '@hawtiosrc/help'
+import help from './help.md'
+import {Springboot} from "@hawtiosrc/plugins/springboot/Springboot"
+
+export const springboot: HawtioPlugin = () => {
+  hawtio.addPlugin({
+    id: pluginId,
+    title: 'Springboot',
+    path: pluginPath,
+    component: Springboot,
+    isActive: async () => workspace.hasMBeans(),
+  })
+  helpRegistry.add(pluginId, 'Runtime', help, 16)
```

Comment:
Note also I recently introduced `order` option in the Plugin API. Loot at other plugin definitions and please follow them. And `16` is already used by Runtime. It should be `17`.

Reply:
@mmelko This part is not yet addressed correctly. Look at other plugins and see what they do.

Reply:
I’ll take care of it.